schema/v1/email-content.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
```

Comment:
@eacarvalho why did we specify a different `content-schema.json` in each GN/channel. The BE always looks at `subject` and `body` for email.

For push is always looks at `title` and `body` so I assume the schema is used by engage app only from the template dir?

Reply:
This `content-schema.json` is imported on Content Enricher and template API returns it and this drives the UI in how to build the Engagement Template on Engage App.

So this is for manageable general notification. Definitely needs an alignment because should be `title` and `body` and not `subject` and `body`.

Reply:
Cool @eacarvalho @ajitavbb lets align tomorrow